docs/storefront/graphql/examples/products.mdx

```diff
@@ -598,6 +598,159 @@ You can query product images at different resolutions. The following query retri
   </Tab>
 </Tabs>
 
+## Get product videos
+
+Query the product videos and all configurable media types for products. The API returns images uploaded in the control panel, image URLs added to a product using the control panel, and video URLs linked to a product. 
+
+The following example retrieves video titles and URLs associated with videos attached to a product. 
```

Comment:
Alternatively, you could use this as the opening line under the header. Because it's more to the point.